src/js/builtins/BundlerPlugin.ts

```diff
@@ -61,6 +61,15 @@ export function loadAndResolvePluginsForServe(
     root: bunfig_folder,
   };
 
+  class InvalidBundlerPluginError extends TypeError {
+    pluginName: string;
```

Comment:
is InvalidBundlerPluginError even catchable here? can the `pluginName` field be read?

i did not check but i suspect this will be overly verbose in the console

```
InvalidBundlerPluginError: "{...}" is not a valid bundler plugin: {...}
  ------------------                        --------------------
   info repeated                            text repeated verbaitim
```

Reply:
I'll make this use `Error` as its `name` property, so users will still see `Error: "foo-plugin" is no a valid plugin: some reason`